src/main/java/main/health/AuthHealthCheck.java

```diff
@@ -0,0 +1,23 @@
+package main.health;
+
+import com.codahale.metrics.health.HealthCheck;
+import twitter4j.Twitter;
+import twitter4j.TwitterFactory;
+import twitter4j.auth.Authorization;
+import twitter4j.auth.NullAuthorization;
+
+public class AuthHealthCheck extends HealthCheck {
+
+    @Override
+    protected Result check() {
+        System.out.println("Checking authorization...");
+        Twitter twitter = TwitterFactory.getSingleton();
+        Authorization auth = twitter.getAuthorization();
+        if (auth.getClass() == NullAuthorization.class){
```

Comment:
why the check for auth here?

Reply:
I mainly added this because Dropwizard spits out an angry message if no health checks are added; checking for authorization seemed like a sensible candidate for a health check. Would you like me to remove the health check?